src/main/java/at/hannibal2/skyhanni/features/mining/crystalhollows/CrystalHollowsLobbyAgeWarning.kt

```diff
@@ -0,0 +1,100 @@
+package at.hannibal2.skyhanni.features.mining.crystalhollows
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.data.jsonobjects.repo.CrystalHollowsLobbyAgeWarningJson
+import at.hannibal2.skyhanni.events.IslandChangeEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.events.RepositoryReloadEvent
+import at.hannibal2.skyhanni.events.TabListUpdateEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.isInIsland
+import at.hannibal2.skyhanni.utils.NumberUtil.formatNumber
+import at.hannibal2.skyhanni.utils.StringUtils
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.TabListData
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraft.client.Minecraft
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.math.abs
+
+class CrystalHollowsLobbyAgeWarning {
+    private val config get() = SkyHanniMod.feature.mining.crystalHollowsLobbyAgeWarning
+    private val mc get() = Minecraft.getMinecraft()
+
+    private val THE_CRYSTAL_HOLLOWS = IslandType.CRYSTAL_HOLLOWS
+
+    private val TICKS_PER_MC_DAY: Long = 24000L
+
+    private val patternGroup = RepoPattern.group("mining.crystalhollows.lobbyage")
+    private val playerCountTabListPattern by patternGroup.pattern(
+        "playercount.tablist",
+        " *(?:§.)*Players[\\S ]*[({\\[](?<playerCount>[\\S ]+)[)}\\]]"
```

Comment:
```suggestion
        "\\s+(?:§.)+Players\\s+(?:§.)+\\((?<playerCount>\\d+)\\)"
```

this should be better